tests/bluetooth/host/id/bt_br_oob_get_local/CMakeLists.txt

```diff
@@ -9,12 +9,12 @@ if (CMAKE_C_COMPILER_ID MATCHES "GNU")
 add_compile_options(-Wno-error=format-zero-length -Wno-format-zero-length)
 endif()
 
+find_package(Zephyr COMPONENTS unittest HINTS $ENV{ZEPHYR_BASE})
```

Comment:
moving this above the `include_directories()` call may change how include directories on targets in unintended ways.

We must be extra careful on this.

That said, generally `include_directories()` should not be needed so we should start by checking if and why `include_directories()` were chosen in this particular case.

Maybe `zephyr_include_directories()` is sufficient in this case.

Reply:
`zephyr_include_directories` does not work because this is doing a unit test and it does not have the named build that `zephyr_include_directories` uses, so I went with the existing `include_directories` and ran it with twister - it passed on my machine and looks like the CI here has passed. @Thalley @jhedberg @cvinayak any comments on this or if it should be changed?

Reply:
I'm not particularly well versed in CMake, so I'd need to spend some time understanding the issues raised by @tejlmand , but that will be a low priority for me, so will take quite some time before I get around to it.

Reply:
> zephyr_include_directories does not work because this is doing a unit test and it does not have the named build that zephyr_include_directories uses

Good point, I didn't consider that.

I looked at the generated `build.ninja` files generated with this PR, and the `build.ninja` files generated in main.
They are identical, thus indicating that `include_directories()` also works on the libraries created as part Zephyr package, although those are specified before the `include_directories()` call.

Thus approving.